changelogs/fragments/support_almalinux.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- Add AlmaLinux to the familiy of Red Hat-alike operating systems
```

I would consider this a bugfix, which will also allow the change to be backported.
```suggestion
bugfixes:
  - add AlmaLinux to fact gathering (https://github.com/ansible/ansible/pull/73458)
```